cli/root.go

```diff
@@ -583,12 +584,17 @@ func checkVersions(cmd *cobra.Command, client *codersdk.Client) error {
 	}
 
 	fmtWarningText := `version mismatch: client %s, server %s
-download the server version with: 'curl -L https://coder.com/install.sh | sh -s -- --version %s'
 `
+	// Our installation script doesn't work on Windows, so instead we direct the user
+	// to the GitHub release page to download the latest installer.
+	if runtime.GOOS != "windows" {
+		fmtWarningText += `download the server version from: https://github.com/coder/coder/releases/tag/v%s`
```

Comment:
This also works, don't know if there's a difference:

```suggestion
		fmtWarningText += `download the server version from: https://github.com/coder/coder/releases/v%s`
```